app/main.py

```diff
@@ -1,6 +1,44 @@
+import os
+
+import requests
+
+URL = "https://api.weatherapi.com/v1/current.json"
+
+API_KEY = os.getenv("API_KEY")
+
+PARAMS = {
+    "key": API_KEY,
+    "q": "Paris",
+}
+
+
+def get_res() -> dict:
+    if not API_KEY:
+        raise ValueError("API key is not set!!!")
+    try:
+        print("Performing request to Weather API to city Paris...")
+        response = requests.get(URL, params=PARAMS)
+        response.raise_for_status()
+        return response.json()
```

Comment:
The `return response.json()` statement is outside the `try` block. If an exception occurs before this line, it could lead to a `NameError`. Consider handling this case appropriately.